CRM/Core/BAO/Navigation.php

```diff
@@ -922,8 +925,9 @@ static public function createOrUpdateTopLevelReportsNavItem($domain_id) {
    */
   public static function getNavItemByUrl($url, $url_params, $parent_id = NULL) {
     $nav = new CRM_Core_BAO_Navigation();
-    $nav->url = "{$url}?{$url_params}";
     $nav->parent_id = $parent_id;
+    $nav->whereAdd("url LIKE '{$url}?{$url_params}'");
```

Comment:
@totten is doing $dao->whereAdd like this 'safe' - the function isn't really exposed to the outside world (ie. the $url & $url_params vars can't be passed in by a user action) - but I still want to check I'm using an acceptable syntax.

Note the change is to allow wildcards in $url_params.